frontend/src/metabase/reference/containers/ReferenceGettingStartedGuide.jsx

```diff
@@ -532,56 +532,53 @@ export default class ReferenceGettingStartedGuide extends Component {
 
                             <div className="mt4 pt4">
                                 <SectionHeader trim={(!has(guide.important_segments) && !has(guide.important_tables))}>
-                                    { Object.keys(segments) > 0 ? 'Segments and tables' : 'Tables' }
+                                    { Object.keys(segments).length > 0 ? 'Segments and tables' : 'Tables' }
```

Comment:
`has` looks like it's different: `export const has = (entity) => entity && entity.length > 0;`